x2gbfs/providers/opendatahub.py

```diff
@@ -7,63 +7,60 @@
 from x2gbfs.gbfs.base_provider import BaseProvider
 
 logger = logging.getLogger(__name__)
+HEADERS = {'User-Agent': 'x2gbfs'}
 
 
-class NoiProvider(BaseProvider):
-    STATION_URL = 'https://mobility.api.opendatahub.com/v2/flat%2Cnode/CarsharingStation?limit=500&offset=0&shownull=false&distinct=true'
-    CAR_URL = 'https://mobility.api.opendatahub.com/v2/flat%2Cnode/CarsharingCar?limit=500&offset=0&shownull=false&distinct=true'
+class OpenDataHubProvider(BaseProvider):
+    STATION_URL = 'https://mobility.api.opendatahub.com/v2/flat,node/CarsharingStation?where=sorigin.eq.%22AlpsGo%22,sactive.eq.true'
+    CAR_URL = 'https://mobility.api.opendatahub.com/v2/tree,node/CarsharingCar/current-station,availability/latest?where=sorigin.eq.%22AlpsGo%22,sactive.eq.true'
```

Comment:
For now, this seems to be sufficient. Later on, you might use url templates and provide the car sharing provider id via the feed's config.

Reply:
Good point. I've added a comment to remind myself.